cmd/edgex-ui-server/res/configuration.yaml

```diff
@@ -8,31 +8,52 @@ Service:
     HealthCheckInterval: 10s
     MaxRequestSize: 0
     RequestTimeout: 5s
+    SecurityOptions:
+        Mode: ""
+        OpenZitiController: "openziti:1280"
 Clients:
     core-data:
         Protocol: http
         Host: localhost
         Port: 59880
+        SecurityOptions:
+            Mode: ""
+            OpenZitiController: "openziti:1280"
```

Comment:
Do we need the Service name here on the clients? Or can we dynamically calculate that using the key which is alway the service key. i.e. `core-data` in this case.